src/core/CCOpt.mli

```diff
@@ -94,6 +98,14 @@ val (<$>) : ('a -> 'b) -> 'a t -> 'b t
 
 (** {2 Alternatives} *)
 
+val or_ : else_:('a t) -> 'a t -> 'a t
+(** [or_ b a] is [a] if [a] is [Some _], [b] otherwise
+    @since NEXT_RELEASE *)
+
+val or_lazy : (unit -> 'a t) -> 'a t -> 'a t
```

Comment:
sorry to nitpick again (almost there), should we add `~else_` to this function, too? For the sake of consistency with `or_`?